src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Base/App.xaml

```diff
@@ -16,19 +16,27 @@
 			<ResourceDictionary.MergedDictionaries>
 				<!-- Load WinUI resources -->
 				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
-
+<!--#if (useDefaultAppTemplate)-->
+<!--#if (useMaterial)-->
 				<MaterialColors xmlns="using:Uno.Material"
-								OverrideSource="ms-appx:///Styles/ColorPaletteOverride.xaml" />
+								OverrideSource="ms-appx:///MyExtensionsApp/Styles/ColorPaletteOverride.xaml" />
```

Comment:
OverrideSource is causing issues with Windows. For the moment, just comment out the overridesource eg

				<MaterialColors xmlns="using:Uno.Material" />
				<!--OverrideSource="ms-appx:///UpdatedUnoExtApp04/Styles/ColorPaletteOverride.xaml" />-->
				<MaterialFonts xmlns="using:Uno.Material" />
				<!--OverrideSource="ms-appx:///UpdatedUnoExtApp04/Styles/MaterialFontsOverride.xaml" />-->